### PR TITLE
Remove task manager components

### PR DESCRIPTION
**Pull request by profileEnumerable**

### CODE REVIEW

File: Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.sass [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "src/styles/styles"
+@import "../../../styles/styles"
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.sass:1]: Use path alias in style preprocessing. Here is an [example](https://stackoverflow.com/questions/42865697/scss-import-relative-to-root)

File: Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div class="sidebar">
+  <div class="sidebar__logo">
+    <img src="../../../assets/img/user/logo.png" alt="logo">
```
Comment (antonAce) [inline Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.html:3]: Use relative path

**Review by antonAce — COMMENTED** [review]
Looks nice so far! Just resolve these minor issues

File: Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.sass (hunk 1 above)
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.sass:1]: Added !

File: Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.html (hunk 2 above)
Comment (profileEnumerable) [reply Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.html]: Simplified !

**Review by antonAce — APPROVED** [review]
:shipit:

**Pull request merged**